app/tests/test_views.py

```diff
@@ -234,6 +234,18 @@ def test_step_3_with_invalid_areaid_is_not_found(self):
         self.assert404(self.client.get('/register/step/3/blah/1'))
 
 class ActivityFeedTests(ViewTestCase):
+    def test_activity_link_present(self):
+        self.login()
+        user = User.query_in_deployment()\
+                 .filter(User.email == 'test@example.org').one()
+        user.first_name = 'John'
+        user.last_name = 'Doe'
```

Comment:
Is there a particular reason you're doing this stuff with the `user` object here?